### PR TITLE
Make project conda installable and CI-testable

### PR DESCRIPTION
**Pull request by jaimergp**
This PR will address the following points:

- [x] Adopt a package-like structure
- [x] Add simple CI to make sure notebooks can run
- [X] Create a conda recipe to specify dependencies
- [ ] Some dependencies are not conda installable yet!
  - [x] `redo` -- submitted to conda-forge [here](https://github.com/conda-forge/staged-recipes/pull/12663)
  - [x] `suds-community` -- used only in the online API examples, since now it's not needed for the online docking parts that are offline
  - [ ] `autodocktools-prepare-py3k` -- same, since we are probably ditching `vina` for `rxdock`, this ugly hack of a package won't be needed
  - [ ] `opencadd` -- this is the trickiest... being a development package it's going to have a lot of updates. I have to think about this a bit... but a `volkamerlab` channel might be involved in the early stages of this deployment.

- [ ] TODO: Update this description

### CODE REVIEW

**Comment by jaimergp** [comment]
@dominiquesydow, just to make sure I am not missing anything, talktorials 013, 014 and 017 are supposed to be missing at this early point, right?

**Comment by dominiquesydow** [comment]
> @dominiquesydow, just to make sure I am not missing anything, talktorials 013, 014 and 017 are supposed to be missing at this early point, right?

Yes, exactly.

For the sake of completeness, I am linking the README containing the talktorial outline (has not obvious "missing" label though):
https://github.com/volkamerlab/TeachOpenCADD/blob/packaging/teachopencadd/README.md

File: teachopencadd/000_template/talktorial.ipynb [hunk 1]
```diff
@@ -161,29 +161,12 @@
...
-   "file_extension": ".py",
-   "mimetype": "text/x-python",
-   "name": "python",
-   "nbconvert_exporter": "python",
-   "pygments_lexer": "ipython3",
-   "version": "3.7.6"
+   "name": ""
   }
  },
  "nbformat": 4,
```
Comment (dominiquesydow) [inline teachopencadd/000_template/talktorial.ipynb]: @jaimergp, have you done this manually for all notebooks or is there an automated way to clear metadata from notebooks?

Comment (jaimergp) [reply teachopencadd/000_template/talktorial.ipynb]: There must be a way but in this case I did it manually after figuring out the click pattern :D

**Comment by dominiquesydow** [comment]
@jaimergp - the README and docs (index/installing) are good to go from my side.

**Comment by dominiquesydow** [comment]
Hi @jaimergp, tests fail because there is a service disruption for the PDB API on November 3-11. This effects our `pypdb` talktorials T008 + T010. I suggest we wait with the merge until after November 11, what do you think?

**Comment by jaimergp** [comment]
Ok, we'll let it settle until tomorrow 🍰

**Comment by jaimergp** [comment]
I'll retrigger the CI now.

**Comment by dominiquesydow** [comment]
Yay, all checks have passed! Let's merge? I let you have the honor :)

**Pull request merged**